tests/test_bidstream_client.py

```diff
@@ -60,6 +60,10 @@ class TestBidStreamClient(unittest.TestCase):
     _CONST_BASE_URL = 'base_url'
     _CONST_API_KEY = 'api_key'
 
+    def refresh(self, refresh_json):
```

Comment:
Should this be called something like `refresh_and_assert_success` to match `_decrypt_and_assert_success`?

Reply:
Hmm..I tried to match C# SDK where its called just `Refresh()`